lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -336,6 +350,86 @@ class CategoryRenderer {
   }
 
   /**
+   * Render a screenshot filmstrip and metrics bars
+   * @param {!Array<!ReportRenderer.AuditJSON>} metricAudits
+   * @param {*} artifacts
+   * @return {?Element}
+   */
+  _renderPerfTimeline(metricAudits, artifacts) {
+    if (!artifacts)
+      return;
+    const tracePass = artifacts['traces'] ? artifacts['traces']['defaultPass'] : null;
+    if (!tracePass)
+      return;
+
+    const fmp = metricAudits.find(a => a.id === 'first-meaningful-paint').result;
+    if (!fmp || !fmp['extendedInfo'])
+      return;
+
+    const tti = metricAudits.find(a => a.id === 'time-to-interactive').result;
+    if (!tti || !tti['extendedInfo'])
+      return;
+
+    const navStart = fmp['extendedInfo']['value']['timestamps']['navStart'];
+    const markers = [
+      {
+        title: 'First contentful paint',
```

Comment:
sgtm, though honestly it may make more sense to keep speed index out of the timeline too since people might be looking for what happened at that point not knowing it's an average